cmd/nitro/nitro.go

```diff
@@ -230,6 +230,8 @@ func mainImpl() int {
 
 	var dataSigner signature.DataSignerFunc
 	var snapshotSigner signature.DataSignerFunc
+	var snapshotAddress *common.Address
```

Comment:
I kept forgetting what this variable was for during my review of this; Maybe we should consider renaming this variable? I think it's not really the snapshot's address, it's the address of the key that we use to sign the snapshots. Perhaps `snapshotSignerAddress`? `snapshotSignatureAddress`?